src/sentry/tasks/scheduler.py

```diff
@@ -0,0 +1,34 @@
+
+from __future__ import absolute_import, division
+
+from celery.utils.log import get_task_logger
+from django.utils import timezone
+
+from sentry.models import ScheduledJob
+from sentry.tasks.base import instrumented_task
+from sentry.utils.cache import Lock, UnableToGetLock
+
+logger = get_task_logger(__name__)
+
+
+@instrumented_task(name='sentry.tasks.enqueue_scheduled_jobs')
+def enqueue_scheduled_jobs(**kwargs):
+    from sentry.celery import app
+
+    lock_key = 'scheduler:process'
+    try:
+        with Lock(lock_key, nowait=True, timeout=60):
+            queryset = list(ScheduledJob.objects.filter(
+                date_scheduled__lte=timezone.now(),
+            )[:100])
```

Comment:
If we had a ton of scheduled jobs, this would be possible that it'd start backing up, since it's only consuming 100 at a time, then waiting 1 minute between.

In practice, I don't think this is an issue since it'll be pretty lightly utilized.

And to be pedantic, this isn't a queryset. ;)